### DISCUSSION TITLE
to_pdf command

### DISCUSSION THREAD

**Discussion by BunkerBuster73** [danielmiessler/Fabric#965] [General]
Hi, installed to_pdf but seems is not working as show in the example.
Anybody have any insight on it ?

what should be the command to create a pdf from a text/clipboard for example ?
i should have Latex Installed and set the path correctly becasue if i prompt
pdflatex i receive 
"This is pdfTeX, Version 3.141592653-2.6-1.40.26 (TeX Live 2024) (preloaded format=pdflatex)
 restricted \write18 enabled."

But using the command to_pdf give a not command recognized
zsh: command not found: to_pdf

I install using the command

go install github.com/danielmiessler/fabric/to_pdf/to_pdf@latest 

but i receive this message

go: github.com/danielmiessler/fabric/to_pdf/to_pdf@latest: module github.com/danielmiessler/fabric@latest found (v1.4.13), but does not contain package github.com/danielmiessler/fabric/to_pdf/to_pdf

Thank you
Bunker

**Comment by mstath1s** [comment]
Same with this: https://github.com/danielmiessler/fabric/issues/956

**Reply by BunkerBuster73** [reply]
oh sorry i didn't see that. thank you.

**Reply by BunkerBuster73** [reply]
oh sorry i didn't see that. thank you.

**Comment by orbenda83** [comment]
I tried the solution on the other ticket and I still have the issue:
[go install github.com/danielmiessler/fabric/to_pdf@latest](go: github.com/danielmiessler/fabric/to_pdf@latest: module github.com/danielmiessler/fabric@latest found (v1.4.88), but does not contain package github.com/danielmiessler/fabric/to_pdf)
